### ISSUE TITLE
Add verified toggle to Admin UI

### ISSUE THREAD

**Issue by alex-fusionauth** [FusionAuth/fusionauth-issues#2897]
### Problem
There has been an API added in #1319 to allow for the verify to be updated, but no way to update in the Admin UI.

### Solution
Add toggle at all necessary levels.

### Alternatives/workarounds
Can only use API.

### Community guidelines
All issues filed in this repository must abide by the [FusionAuth community guidelines](https://fusionauth.io/community/forum/topic/1000/code-of-conduct).

### How to vote
Please give us a thumbs up or thumbs down as a reaction to help us prioritize this feature. Feel free to comment if you have a particular need or comment on how this feature should work.

**Comment by jobannon** [comment]
@alex-fusionauth We currently expose a toggle to verify the email in one direction.
<img width="2537" alt="image" src="https://github.com/user-attachments/assets/9a245d62-a2ed-4424-a2ac-cd6f2851a352">
What's not exposed is registration verification (updating or modifying that).  I am not sure if our API's allow modification on that layer...

**Comment by alex-fusionauth** [comment]
@jobannon  I believe part of the problem is that when we import users and this is marked we would like to make it `verified: false` in the UI.

**Comment by jobannon** [comment]
Good point.  We should also consider toggles for registration verification status too, as I bet they have the same condition.

**Comment by robotdan** [comment]
Can you clarify the specific request? 

Is the specific request to change a user email verification state to `false` when it is already `true`? If so, what is the use case for this?

**Comment by alex-fusionauth** [comment]
The usecase is when importing incorrectly and you want a user to be forced back to unverified, requiring the flow once again.
